src/core/thread/mle.hpp

```diff
@@ -639,14 +639,11 @@ class Mle : public InstanceLocator, private NonCopyable
 #endif
 
     /**
-     * Requests MLE layer to prepare and send a shorter version of Child ID Request message by only
-     * including the mesh-local IPv6 address in the Address Registration TLV.
-     *
-     * Should be called when a previous MLE Child ID Request message would require fragmentation at 6LoWPAN
-     * layer.
+     * Notifies MLE whether the Child ID Request message was transmitted successfully.
+     * @param[in]  aMessage  The transmitted message.
```

Comment:
```suggestion
     * Notifies MLE whether the Child ID Request message was transmitted successfully.
     *
     * @param[in]  aMessage  The transmitted message.
```

Reply:
Sure, thanks @jwhui, incorporated the change.